src/sources/prometheus/scrape.rs

```diff
@@ -60,6 +61,13 @@ pub struct PrometheusScrapeConfig {
     #[configurable(metadata(docs::human_name = "Scrape Interval"))]
     interval: Duration,
 
+    /// The timeout for each scrape request, in seconds.
+    #[serde(default = "default_target_timeout")]
+    #[serde_as(as = "serde_with::DurationSeconds<u64>")]
```

Comment:
Same suggestion as above.
```suggestion
    #[serde_as(as = "serde_with:: DurationSecondsWithFrac<f64>")]
```